spec/shared/cassette_secrets_helper.rb

```diff
@@ -0,0 +1,39 @@
+module Spec
+  module Shared
+    module CassetteSecretsHelper
+      def default_vcr_secrets_path
+        Pathname.new(Dir.pwd).join("config/secrets.defaults.yml").tap do |path|
+          raise "Default vcr cassette secrets not found: #{path}! Create this file with placeholder secrets to be used in cassettes to avoid leaking actual secrets." unless path.exist?
+        end
+      end
+
+      def vcr_secrets_path
+        Pathname.new(Dir.pwd).join("config/secrets.yml")
+      end
```

Comment:
My other comment got moved away, but I'd prefer:

```suggestion
      def default_vcr_secrets_path
        root = defined?(ENGINE_ROOT) ? Pathname.new(ENGINE_ROOT) : Rails.root
        root.join("config/secrets.defaults.yml").tap do |path|
          raise "Default vcr cassette secrets not found: #{path}! Create this file with placeholder secrets to be used in cassettes to avoid leaking actual secrets." unless path.exist?
        end
      end

      def vcr_secrets_path
        root = defined?(ENGINE_ROOT) ? Pathname.new(ENGINE_ROOT) : Rails.root
        root.join("config/secrets.yml")
      end
```

Reply:
yeah, this is sufficiently buried behind other priorities so it will not be for a while